.github/workflows/hugo.yaml

```diff
@@ -51,30 +51,36 @@ jobs:
       if: github.event_name == 'release' && github.event.action == 'published'
       id: version
       run: |
-        COMMIT_SHA="${GITHUB_SHA}"
         if [[ $GITHUB_REF == refs/tags/* ]]; then
           TAGS=${GITHUB_REF#refs/tags/}
+        else
+          TAGS="none"
         fi
         echo ::set-output name=tags::${TAGS}
     - name: "Build Versioned Hugo Release"
       if: github.event_name == 'release' && github.event.action == 'published'
       env:
-        HUGO_URL: ${HUGO_URL}release/${{ steps.version.outputs.tags }}
+        HUGO_URL: ${{ env.HUGO_URL }}release/${{ steps.version.outputs.tags }}
         HUGO_PUBLISH_DIR: public/release/${{ steps.version.outputs.tags }}
+        SEMVERSION_TAG: ${{ steps.version.outputs.tags }}
       run: |
         make lint
         make release
+        sudo chown -R $(whoami):$(whoami) ./themes/cloudposse/node_modules/
         make real-clean hugo/build
     - name: "Push Site to S3"
       if: github.event_name == 'release' && github.event.action == 'published'
       env:
         S3_BUCKET_NAME: docs.prod.cloudposse.org
         AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        HUGO_URL: ${{ env.HUGO_URL }}release/${{ steps.version.outputs.tags }}
+        SEMVERSION_TAG: ${{ steps.version.outputs.tags }}
       run: make deploy
     - name: "Push New Index to Algolia"
       if: github.event_name == 'release' && github.event.action == 'published'
       env:
+        HUGO_PUBLISH_DIR: public/release/${{ steps.version.outputs.tags }}
```

Comment:
I think this is the wrong index file. We don't want search pointing to the versioned release. We want it  pointing to what's deployed in /

/release/x.y.z is for historical archives

Reply:
This was resolved by removing that line, which defaults `HUGO_PUBLISH_DIR` to `public`, the non-versioned website build.